source/_UIAHandler.py

```diff
@@ -748,19 +748,27 @@ def _isUIAWindowHelper(self,hwnd):
 		if res:
 			# The window does support UIA natively, but MS Word documents now
 			# have a fairly usable UI Automation implementation.
-			# However, builds of MS Office 2016 before build 9000 or so had bugs which
+			# However, builds of MS Office 2016 before build 13901 or so had bugs which
 			# we cannot work around.
-			# And even current builds of Office 2016 are still missing enough info from
-			# UIA that it is still impossible to switch to UIA completely.
-			# Therefore, if we can inject in-process, refuse to use UIA and instead
+			# Therefore for less recent versions of Office,
+			# if we can inject in-process, refuse to use UIA and instead
 			# fall back to the MS Word object model.
 			canUseOlderInProcessApproach = bool(appModule.helperLocalBindingHandle)
+			isOfficeApp = appModule.productName.startswith(("Microsoft Office", "Microsoft Outlook"))
 			if (
-				# An MS Word document window 
-				windowClass=="_WwG" 
+				(
+					winVersion.getWinVer() < winVersion.WIN10
+					# An MS Office app before build 13901
+					or (
+						isOfficeApp
+						and int(appModule.productVersion.split('.')[2]) < 13901

```

Comment:
Should this check include the `major.minor` aspects of the build version? Eg:

```suggestion
						and int(appModule.productVersion.split('.')[2]) < 13901
						and int(appModule.productVersion.split('.')[0]) == 16
						and int(appModule.productVersion.split('.')[1]) == 0
```